client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -219,11 +221,15 @@ class SimplePaymentsDialog extends Component {
 					.addPost( productToCustomPost( productForm ) )
 					.then( newProduct => {
 						dispatch( receiveUpdateProduct( siteId, customPostToProduct( newProduct ) ) );
+						analytics.tracks.recordEvent( 'calypso_simple_payments_button_create', {
+							price: productForm.price,
+							currency: productForm.currency,
```

Comment:
I don't think we have a way of tracking "get events that have matching ids". But I see no harm, so I'll add